<?xml version="1.0" encoding="UTF-8"?>

<launch>

<node name="controller" pkg="pid" type="controller" ns="sweep_pid" output="screen" >
      <param name="node_name" value="sweep_pid" />
      <param name="Kp" value="5.0" />
      <param name="Ki" value="0.0" />
      <param name="Kd" value="0.1" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
     </node>

 <node name="PID" pkg="robotic_arm_quark" type="PID.py" output="screen"/>
 <!-- <node name="masterController" pkg="robotic_arm_quark" type="masterController.py" output="screen"/> -->
 <node name="CV" pkg="robotic_arm_quark" type="CV.py" output="screen"/>

  <include file = "$(find robotic_arm_quark)/launch/load_param.launch">
  </include>



</launch>